code/modules/vore/eating/digest_act_vr.dm

```diff
@@ -0,0 +1,88 @@
+//Please make sure to:
+//return FALSE: You are not going away, stop asking me to digest.
+//return TRUE: You are going away.
+
+// Ye default implementation.
+/obj/item/proc/digest_act(var/datum/belly/belly = null)
+	if(belly)
+		for(var/obj/item/AM in contents)
+			AM.forceMove(belly.owner)
+			belly.internal_contents |= AM
+
+		belly.owner.nutrition += (1 * w_class)
+		if(isrobot(belly.owner))
+			var/mob/living/silicon/robot/R = belly.owner
+			R.cell.charge += (50 * w_class)
+	qdel(src)
```

Comment:
Needs borg sleeper support (borg sleeper is not a vorepanel belly) or everything will just nuke along with the possible contents for zero gains.